portalnet/src/overlay/service.rs

```diff
@@ -112,7 +113,7 @@ where
     /// The routing table of the local node.
     kbuckets: Arc<RwLock<KBucketsTable<NodeId, Node>>>,
     /// The protocol identifier.
-    protocol: ProtocolId,
+    protocol: Subnetwork,
```

Comment:
I left the original `protocol` var alone here since it's used extensively in our logs & comments. At some point, these two phrases are interchangeable, so i'm not sure it's worth updating all the `protocol` usage in our comments to `subnetwork`

Reply:
I think it is fine to leave this as you did